contents/articles/2019-03_vizard-plugin/index.md

```diff
@@ -0,0 +1,34 @@
+--- 
+title: New 200Hz Vive add-on and WorldViz Vizard support
+date: Tue Mar 20 2019 17:25:31 GMT+0700 (DST) 
+author: Pupil Dev Team 
+subtitle: Firstly we are pleased to announce our new HTC Vive 200Hz add-on and WorldViz Vizard 6 support...
```

Comment:
We are pleased to announce our new HTC Vive/Vive PRO 200Hz add-on and WorldViz Vizard 6 support for Pupil...